proto/hack/generate_proto

```diff
@@ -10,7 +10,7 @@ CONTROL_PLANE_OUTPUT_DIR=./
 mkdir -p $DATA_PLANE_OUTPUT_DIR
 mkdir -p $CONTROL_PLANE_OUTPUT_DIR
 
-docker run --rm -v "${PWD}":"${PWD}" --security-opt label:disable -w "${PWD}" jaegertracing/protobuf:latest \
+docker run --rm -v "${PWD}":"${PWD}" --security-opt label:disable -w "${PWD}" thethingsindustries/protoc:latest \
```

Comment:
Is it better? I'm not sure why...

Reply:
Because it properly generates golang code with the newest google package